c2corg_ui/templates/helpers/view.html

```diff
@@ -215,8 +188,25 @@ <h3 class="heading show-phone" ng-click="mainCtrl.animateHeaderIcon($event)" dat
         <div class="list-item" ng-repeat="doc in ${associations}" ng-class="{'new-item': doc['new']}">
           <app-card app-card-doc="doc"></app-card>
           % if type == 'articles':
-            <app-delete-association child-doctype="articles" parent-id="${document['document_id']}" child-id="doc.document_id"
-              ng-if="userCtrl.auth.isAuthenticated()"></app-delete-association>
+              <app-delete-association child-doctype="articles" parent-id="${document['document_id']}" child-id="doc.document_id"
+                ng-if="userCtrl.auth.isAuthenticated()
+              % if is_collaborative(document):
+                  && userCtrl.auth.isModerator()
+
+              % elif is_personal(document):
+                  && userCtrl.auth.isModerator()
+                  || userCtrl.hasEditRights
```

Comment:
> As you suggest it, it makes sense to remove ``userCtrl.auth.isModerator()`` here since ``userCtrl.hasEditRights`` already test for moderators.

In addition it would the test a bit less confusing. Because 
```
userCtrl.auth.isAuthenticated() && userCtrl.auth.isModerator() || userCtrl.hasEditRights()
```
probably means "(isAuth AND isModerator) OR hasEditRights".
OK it would work because "hasEditRights" also tests if "isModerator" (and "isAuth"?). But that looks like a potentially buggy condition...
Same problem for the other conditions a bit further.